### PR TITLE
Fixing the PIP mode on iOS

### PR DESCRIPTION
**Pull request by lrusso**
#### Describe the changes

Fixing the PIP mode on iOS. It was not working when using the `pictureInPicture` property, nothing happened. Also, I fixed the `onPictureInPictureStatusChanged` event because it was never called.

If you are using a simulator, please test this PR on an iPad simulator because the iPhone simulator doesn't have the PIP mode available.

Fix for issues like: https://github.com/react-native-video/react-native-video/issues/3131

Based on the work of: https://github.com/react-native-video/react-native-video/pull/3016

### CODE REVIEW

File: ios/Video/Features/RCTPlayerOperations.swift [hunk 1]
```diff
@@ -112,15 +112,13 @@ enum RCTPlayerOperations {
...
-                // Do noting. Fix for tvOS native audio menu language selector
-            #else
+            #if os(iOS)
                 player?.currentItem?.selectMediaOptionAutomatically(in: group)
                 return
             #endif
         }
 
-        #if os(tvOS)
+        #if os(iOS)
```
Comment (freeboub) [inline ios/Video/Features/RCTPlayerOperations.swift]: Are you sure of this change ?

**Comment by freeboub** [comment]
@lrusso pleas update on the fix, I would like to merge it soon.
Can it fix this issue ? https://github.com/react-native-video/react-native-video/issues/3069
Thank you!

**Comment by freeboub** [comment]
and also this one: https://github.com/react-native-video/react-native-video/issues/3131 ?

**Comment by lrusso** [comment]
hello @freeboub #3131 is fixed with this pull request, and it does not fix https://github.com/react-native-video/react-native-video/issues/3069 because is not related to it.

**Comment by freeboub** [comment]
@lrusso can you just check this comment please https://github.com/react-native-video/react-native-video/pull/3221#pullrequestreview-1603989760

**Comment by lrusso** [comment]
@freeboub can you rephrase please? what comment should I check? I just checked the issue https://github.com/react-native-video/react-native-video/issues/3131 and it is solved with this pull request and https://github.com/react-native-video/react-native-video/issues/3069 is not related to this fix.

File: ios/Video/Features/RCTPlayerOperations.swift (hunk 1 above)
Comment (freeboub) [reply ios/Video/Features/RCTPlayerOperations.swift]: @lrusso Sorry I didn't click on finish review :'( 
I am talking about this comment

Comment (lrusso) [reply ios/Video/Features/RCTPlayerOperations.swift]: you are right, I just committed a rollback for tvOS.

**Comment by freeboub** [comment]
Thank you @lrusso !
